### ISSUE TITLE
Suggestions For CS2

### ISSUE THREAD

**Issue by accelerate0** [PenguiniVogel/BuffUtility#42]
Due to how the only way to inspect skins for cs2 is through  `csgo_econ_action_preview `, it would be helpful to add the floatdb compare feature on the favorite list and expanded detail list as well in order to inspect skins on CS2.

**Comment by PenguiniVogel** [comment]
Yes 2.2.3 will have inspect link generation, it has been prototyped already
